ultra/train.py

```diff
@@ -19,7 +19,8 @@
 # LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
 # OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 # THE SOFTWARE.
-import os
+import os, sys
+import ijson
```

Comment:
Maybe reading the JSON for the agents can be done with Python's built-in JSON package? Just to avoid another dependency? I'm not sure, what are your thoughts @JenishPatel99 and @AlexLewandowski ?

Reply:
I was familiar the functions that were needed to extract those keys from the file using ijson, I was not sure about the main JSON package. I will take a look into that